db/batchlog_manager.cc

```diff
@@ -94,22 +94,22 @@ db::batchlog_manager::batchlog_manager(cql3::query_processor& qp, batchlog_manag
 future<> db::batchlog_manager::do_batch_log_replay() {
     // Use with_semaphore is much simpler, but nested invoke_on can
     // cause deadlock.
-    return get_batchlog_manager().invoke_on(0, [] (auto& bm) {
```

Comment:
Changelog lacks motivation.

Reply:
I'll send it separately as a followup.
It's immaterial to this fix...